source/features/bypass-checks.tsx

```diff
@@ -7,7 +7,13 @@ import * as api from '../github-helpers/api';
 import {getRepoURL} from '../github-helpers';
 
 async function bypass(detailsLink: HTMLAnchorElement): Promise<void> {
-	const runId = new URLSearchParams(detailsLink.search).get('check_run_id') ?? detailsLink.pathname.split('/').pop()!;
+	const [actionType, splitRunid] = detailsLink.pathname.split('/').splice(-2);
+	const runId = new URLSearchParams(detailsLink.search).get('check_run_id') ?? splitRunid;
+
+	if (!Number.parseInt(runId, 10) || actionType !== 'run') {
```

Comment:
A bit verbose. Can pageDetect help?

Reply:
I am 100% at fault here. I wrote this half in my sleep, I should have waited for the morning...

I will update it later with more details